modules/Bio/EnsEMBL/Production/Search/JSONReformatter.pm

```diff
@@ -58,6 +58,7 @@ sub process_json_file {
   {
     local $/ = '}';
     while (<$fh>) {
+      s/\\\\//g;	    
```

Comment:
I am not sure I understand well your comment, is the issue related to test content? with a hidden char inside? 
Since those test files are issued from test sample DB, is it not safer to fix the sample data instead?

Reply:
There is an unexpected escape character '\' in dictionary  { "hidden_remark":"formerly known as \\", "auth…}, which causes the failure, Not sure why test data differs from the Perl version. ya, it's safer to edit the test data.

Reply:
Cheers! let's do that then. Thanks!

Reply:
but I believe we can keep this clean up, just in case some data are containing this nasty char (I think the data are issued from actual db fetched to generated the test content), so it might be best to keep the change, and fix the original DB. to stay on the safest side.

Reply:
sure we will fix the actual data

Reply:
Can you point me to the test file? so we can act on the Test/Live DB to fix this?

Reply:
t/41-searchdumps_reformat_advsearch.t

Reply:
Well, that's the test itself, not the related test datasource. :-)